quodlibet/docs/development/plugins.rst

```diff
@@ -66,7 +66,7 @@ Creating a new Plugin
 
     class MyPlugin(EventPlugin):
         PLUGIN_ID = "myplugin"
-        PLUGIN_NAME = _("My Plugin")
+        PLUGIN_NAME = "My Plugin"
```

Comment:
We've removed the "_" function from the builtins some time ago and you now have to import it explicitly: ``from quodlibet import _``. But I guess it makes sense to keep the example simple and not involve translations.